distributed/tests/test_utils_test.py

```diff
@@ -225,3 +227,27 @@ def test_tls_cluster(tls_client):
 async def test_tls_scheduler(security, cleanup):
     async with Scheduler(security=security, host="localhost") as s:
         assert s.address.startswith("tls")
+
+
+from distributed.core import Status
```

Comment:
Nitpick: Can you move this import to the top of the module with existing `distributed.core` import?

Reply:
sure, I'm somehow used for either linters or formatters to make me aware of this. for some reason our config considers this perfectly fine 🤷‍♂️